src/mpid/ch4/src/ch4_init.c

```diff
@@ -99,7 +99,7 @@
       verbosity   : MPI_T_VERBOSITY_USER_BASIC
       scope       : MPI_T_SCOPE_LOCAL
       description : >-
-        Sets the number of VCIs that user needs (should be a subset of MPIDI_CH4_NUM_VCIS).
+        Sets the number of VCIs that user needs (should be a subset of MPIDI_CH4_MAX_VCIS).
```

Comment:
Hmm, this change crept into this commit. It should be in the previous commit?

Reply:
Right. Rebase is messy :) I'll clean it up.

Reply:
done.